### PR TITLE
Docs polishing

### LINKED ISSUE #104: rename plot_circulareegtopoplot to smth more senseful
for instance:
- topoplot_circular
- circular_topoplot

### LINKED ISSUE #148: titiles doesn't work in topoplot series and parallel plots
I tried this line in plot_toposeries and plot_parallelplots, but haven't seen any effect:
`axis = (; title = "test")`
While in plot_topoplot it works fine.

### PR DESCRIPTION
**Pull request by vladdez**
- unified output
- new plots in docs
- fixed #104 
   -  plot_circulareegtopoplot renamed to plot_circular_topoplots
- fixed #148 
- new feature: xlabel will change from "Trials" to "Trials sorted" in plot_erpimage
- code expanding in docs added
- unified function configs in docstrings (important!)
- line breaks in docstring -> tidy docstrings
- erpimage in docs shown as normalised
- easier way to make titles in docs
- feature for adding title was added for toposeries and parplots

### CODE REVIEW

File: src/plot_erpimage.jl [hunk 1]
```diff
@@ -52,6 +57,7 @@ function plot_erpimage!(
             sortindex = 1:size(plot, 2)
         else
             sortindex = sortperm(sortvalues)
+            ax.ylabel = "Trials sorted" #by $(mymacro(sortvalues))
```
Comment (behinger) [inline src/plot_erpimage.jl:60]: this should be specified via config, else it is not customizable

**Review by behinger — CHANGES_REQUESTED** [review]
very nice! Once you fixed the three small things, ready to merge

File: src/plot_erpimage.jl (hunk 1 above)
Comment (vladdez) [reply src/plot_erpimage.jl:60]: the idea is that in config there is already "Trials"
While here is changed to "Trials sorted"
I also wanted to include by which variable it is sorted

Comment (behinger) [reply src/plot_erpimage.jl:60]: mh, but your solution makes it impossible to overwrite that label via a custom `axis=(;ylabel="My Label is better")`

You could check whether in the kwargs axis.title is empty or not and then use your default maybe

File: src/plot_erpimage.jl [hunk 2]
```diff
@@ -57,7 +56,9 @@ function plot_erpimage!(
             sortindex = 1:size(plot, 2)
         else
             sortindex = sortperm(sortvalues)
-            ax.ylabel = "Trials sorted" #by $(mymacro(sortvalues))
+            if ax.ylabel[] == "Trials"
```
Comment (behinger) [inline src/plot_erpimage.jl:59]: if isnothing(sortindex) && !isnothing(sortvalues)
   config_kwargs!(config;axis=(;ylabel="Trials sorted")
end

Comment (vladdez) [reply src/plot_erpimage.jl:59]: gosh, that was super easy

**Pull request merged**